src/AeroShield.h

```diff
@@ -0,0 +1,64 @@
+/*
+  API for the AeroShield hardware.
+  
+  The file is a part of the application programming interface for
+  the AeroShield didactic tool for control engineering and 
+  mechatronics education.
+  
+  This code is part of the AutomationShield hardware and software
+  ecosystem. Visit http://www.automationshield.com for more
+  details. This code is licensed under a Creative Commons
+  Attribution-NonCommercial 4.0 International License.
+
+  Created by .... 
+  Last update: 02.03.2022.
+*/
```

Comment:
Say what it does, etc. like in the other files.

Reply:
the last meme is super cool :DD I wasnt communicating for the last 2 weeks.... so tomorrow I have a meeting with anka :) We gonna figure it out

Reply:
All right, well has that happened?:)